operating-systems.tex

```diff
@@ -8,6 +8,7 @@ \chapter{Operating Systems}
 the scheduling and resource sharing policies and mechanisms available, with a focus on temporal isolation,
 asymmetric protection, policy freedom, and resource sharing.  
 % TODO is asymmetric protection still a thing we want to support?
+
```

Comment:
I don't know if there was meant to be a line break here that the comment was unintentionally negating.